README.md

```diff
@@ -1 +1,159 @@
-# MDoc
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+  <h1>Doctor Appointment</h1>
+
+</div>
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [Schema](#ER-diagram)
+  - [🛠 Built With ](#-built-with-)
+  - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [Kanban Board](#kanban)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Run Tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+- [🔭 Future Features:](#-future-features)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# MDoc <a name="about-project"></a>
+
+It's an Online Doctor Reservation web application designed to provide users with the ability to reserve, view, and manage doctors and doctor appointment reservation . The system aims to streamline the process of finding and booking doctors from various locations
+
+#### Link to [Frontend Repository](https://github.com/siddghosh108/m-doc)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- User Access: The system allows users to create accounts and log in
+- Doctor reservation: hospitals can list their available doctors. Each listing includes details such as doctors name, date, and location.
+- Reservation Management: Users can view available doctors  at different locations and make reservations
+- Add and delete doctors in including location
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://capstone-m-doc.onrender.com/">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎬 Kanban Board <a id="kanban"></a>
+
+Link to the [Kanban Board](https://github.com/users/prg-04/projects/5)
+
+All 3 Authors have been contributing to this project from start to finish
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Installed Git.
+- Installed Ruby on Rails
+- Installed PostgreSQL
+- Create a local directory that you want to clone the repository.
+- Open the command prompt in the created directory.
+- On the terminal run this command git clone https://github.com/prg-04/MDoc.git
+
+### Setup
```

Comment:
- [ ]  Kindly, include a readme section called "tests" with specific instructions about how to setup and run them correctly. All 31 of them are passing in this review iteration.

![image](https://github.com/prg-04/MDoc/assets/57421823/0f73f20a-3857-4c3f-8df4-aa7a8bfb1359)


That way, your users will be able to test your project's integrity. 👍